credential/store/src/main/java/org/wildfly/security/credential/store/_private/ElytronMessages.java

```diff
@@ -140,4 +140,16 @@ CredentialStoreException invalidCredentialStoreEntryType(Class<? extends KeyStor
     @Message(id = 12000, value = "The credential store file %s does not exist or cannot be accessed.")
     CredentialStoreException credentialStoreFileDoesNotExist(String fileLocation);
 
+    @Message(id = 12001, value = "The credential store has not been initialised.")
```

Comment:
I do not understand the numbering system, but it is suspicious that 12001 and 12002 are already in use, see https://github.com/wildfly-security/wildfly-elytron/blob/1.x/audit/src/main/java/org/wildfly/security/audit/ElytronMessages.java#L71 and https://github.com/wildfly-security/wildfly-elytron/blob/1.x/ELY_Messages.txt#L7

Reply:
The error itself is thrown from any attempt to use the PropertiesCredentialStore before the initialise method has been called so it will be thrown in the context of those calls.

Reply:
Good catch on the message IDs - looking at the history it looks like we didn't pick up on a module only reserving a subset of message IDs when they were supposed to reserve a complete block.

We actually have two definitions of 12000 in the project - I think I will just have to renumber all the credential store numbers from 12000 into a new range.

Reply:
https://issues.redhat.com/browse/ELY-2087

Reply:
@OndrejKotek I have added a commit to clean up the error code.  As a number of releases have gone out with the duplicate I think it is cleaner to just replace both with a new number and take the old duplicated number out of use.

Reply:
Thank you for the fix, this can be resolved.